.github/workflows/tests.yml

```diff
@@ -26,31 +26,50 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
+
     - name: Set up Python ${{ matrix.python }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python }}
+
     - name: Install
       run: |
         python -m pip install --upgrade pip
         python -m pip install -e ".[tests,scripts]" --use-deprecated=legacy-resolver
+
     - name: Print environment
       run: |
         python -m pip freeze
         python --version
         python -c "import pysparkling; print(pysparkling.__version__)"
+
+    - name: Optimize imports
+      run: |
+        isort .
+
+    - uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
I am against auto-commits. If we want to run isort, it should be a check only.

Reply:
Ok, and I thought it was so nifty :-)